universal-login-relayer/lib/core/services/messages/MessageStatusService.ts

```diff
@@ -0,0 +1,28 @@
+import {Wallet} from 'ethers';
+import {MessageStatus} from '@universal-login/commons';
+import IMessageRepository from './IMessagesRepository';
+import {getRequiredSignatures} from '../../utils/utils';
+
+export class MessageStatusService {
+  constructor(private messageRepository: IMessageRepository, private wallet: Wallet) {
+  }
+
+  async getStatus(messageHash: string) {
+    const message = await this.messageRepository.get(messageHash);
+    const required = await getRequiredSignatures(message.walletAddress, this.wallet);
```

Comment:
1) Move to: `integration/utils/WalletService.ts`
2) class WalletService.getRequiredSignatures = (address: string) => 
3) Inject in constructor:
`constructor(..., walletService: WalletService) `